package.json

```
@@ -3,10 +3,11 @@
   "scripts": {
     "build": "parcel build index.html && echo && parcel build --target node src/entry-point-static/build.tsx && echo && node dist/build.js && rm -f dist/build.js dist/build.js.map",
     "clean": "rm -rf dist && rm -rf .cache",
-    "dev": "bin/parcel serve index.html",
-    "build": "parcel build index.html && echo && parcel build --target node src/entry-point-static/build.tsx && echo && node dist/build.js && rm -f dist/build.js dist/build.js.map"
+    "dev": "bin/parcel serve index.html"
   },
   "dependencies": {
+    "@types/ndarray": "^1.0.7",
+    "ndarray": "^1.0.19",
```

Can we use the exact version for these?

```suggestion
    "@types/ndarray": "1.0.7",
    "ndarray": "1.0.19",
```

---

sorry keep forgetting 😬